linux/mount.go

```diff
@@ -570,6 +570,28 @@ func MountDeviceWithFileSystem(devPath string, mountPoint string, options []stri
 	return mount, nil
 }
 
+func MountNFSShare(source string, targetPath string, options []string) error {
+	log.Tracef(">>>>> MountNFSShare called with source %s target %s", source, targetPath)
+	defer log.Tracef("<<<<< MountNFSShare")
+
+	args := []string{source, targetPath}
+	optionArgs := []string{}
+	if len(options) != 0 {
+		optionArgs = append([]string{"-o"}, strings.Join(options, ","))
```

Comment:
should we validate or restrict it to the ones we tested or want to keep it opaque?

Reply:
yes, it should be opaque. we have nfsMountOptions parameter to tweak these. But we can document any options that doesn't work well.